py/datastore_service.py

```diff
@@ -103,11 +104,44 @@ def handle_request(self, post):
     #get the reporter data
     reports = self.parse_reports(post)
 
-    #TODO: build a query
-    #for report in reports:
+    try:
+      # check and see if prepared statement exists...if not, create it
```

Comment:
yeah it is easy to do. just move it into the function called `make_locals` and be sure to save the stuff you need to `self.thing_you_need` then you can access it in the request handler via `self.server.thing_you_need` as we discussed earlier